templates/flavors/machinepool-windows/kustomization.yaml

```diff
@@ -1,9 +1,8 @@
 namespace: default
 resources:
-  - ../base-windows
-  - ../../azure-cluster-identity
-  - machine-pool-deployment.yaml
+  - ../machinepool
   - machine-pool-deployment-windows.yaml
 
 patchesStrategicMerge:
-  - ../../azure-cluster-identity/azurecluster-identity-ref.yaml
+  - ../base-windows-containerd/kubeadm-control-plane.yaml
```

Comment:
should we rename this to just `base-windows`?

Reply:
should we do this as part of https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2315